doc/porting/cmsis_rtos_v2.rst

```diff
@@ -0,0 +1,85 @@
+.. _cmsis_rtos_v2:
+
+CMSIS RTOS v2
+##########################
+
+Cortex-M Software Interface Standard (CMSIS) RTOS is a vendor-independent
+hardware abstraction layer for the ARM Cortex®-M processor series and defines
+generic tool interfaces. Though it was originally defined for ARM Cortex-M
+microcontrollers alone, it could be easily extended to other microcontrollers
+making it generic. For more information on CMSIS RTOS v2, please refer
+http://www.keil.com/pack/doc/CMSIS/RTOS2/html/index.html
```

Comment:
```suggestion
to the `CMSIS-RTOS2 Documentation <http://www.keil.com/pack/doc/CMSIS/RTOS2/html/index.html>`_.
```

Reply:
@dbkinder: This will be addressed in @rgundi's PR (#10954).